src/brsTypes/components/RoDeviceInfo.ts

```diff
@@ -191,8 +207,7 @@ export class RoDeviceInfo extends BrsComponent implements BrsValue {
             returns: ValueKind.String,
         },
         impl: (_interpreter) => {
-            let countryCode = process.env.LOCALE;
-            return countryCode ? new BrsString(countryCode) : new BrsString("");
+            return new BrsString(RoDeviceInfo.locale);
```

Comment:
Should this and below need to be `RoDeviceInfo._locale`?

Reply:
Nope, I think that we want to use the `locale` alias, so that it gets the same treatment of using `process.env.LOCALE` as a back-up value.